Terminal.Gui/View/Layout/ViewLayout.cs

```diff
@@ -209,6 +210,13 @@ public virtual Rect Bounds
                                 );
             }
 #endif // DEBUG
+            if (ScrollBarType != ScrollBarType.None && UseNegativeBoundsLocation)
```

Comment:
You are right. I think it's better to remove the `UseNegativeBoundsLocation` property and the user is free to change the location and dimension as he sees fit. Do you agree?

Reply:
I go back on the idea of removing the `UseNegativeBoundsLocation` property because it needs to be used in views, such as `TextView`, `ListView`, `TableView`, etc... that use their own scrolling mechanism. Only views that do not handle location scrolling can use this property.